packages/graph/src/votingStrategy/quadraticFunding/implementation.ts

```diff
@@ -1,42 +1,52 @@
 import { log } from "@graphprotocol/graph-ts";
 import { Voted as VotedEvent } from "../../../generated/QuadraticFundingVotingStrategy/QuadraticFundingVotingStrategyImplementation";
 import { Round, QFVote, VotingStrategy } from "../../../generated/schema";
+import { generateID } from "../../utils";
 
 /**
  * @dev Handles indexing on Voted event.
  * @param event VotedEvent
  */
 export function handleVote(event: VotedEvent): void {
-  if (!event.receipt) {
-    log.warning("--> handleVote {} : event.receipt is null", ["QF"]);
-    return;
-  }
 
   // load voting strategy contract
   const votingStrategyAddress = event.address;
   let votingStrategy = VotingStrategy.load(votingStrategyAddress.toHex());
+
   if (!votingStrategy) {
-    log.warning("--> handleVotingContractCreated {} : votingStrategy is null", [
+    log.warning("--> handleVotingContractCreated {} {}: votingStrategy is null", [
       "QF",
+      votingStrategyAddress.toHex()
     ]);
     return;
   }
 
-  // load Round contract
-  let round = Round.load(votingStrategy.round!);
-  if (!round) {
-    log.warning("--> handleVotingContractCreated {} : round is null", [
-      "QF",
-    ]);
-    return;
-  }
+  // if (!votingStrategy.round) {
```

Comment:
This is something we have to figure out how to implement

Reply:
See comment above. I think we can get this merged and write out a separate story for the issue here.